fe/src/main/java/org/apache/doris/analysis/LiteralExpr.java

```diff
@@ -45,6 +45,9 @@ protected LiteralExpr(LiteralExpr other) {
     public static LiteralExpr create(String value, Type type) throws AnalysisException {
         Preconditions.checkArgument(type != Type.INVALID);
         LiteralExpr literalExpr = null;
+        if (value.toUpperCase().equals("MAXVALUE")) {
```

Comment:
Can not do this!
`MAXVALUE` should be a keyword of syntex, NOT a keyword of LiteralExpr.
What if user create a string "MAXVALUE"?